src/khoj/routers/api.py

```diff
@@ -586,9 +588,9 @@ async def chat_options(
     return Response(content=json.dumps(cmd_options), media_type="application/json", status_code=200)
 
 
-@api.post("/speak")
+@api.post("/transcribe")
 @requires(["authenticated"])
-async def transcribe_audio(request: Request, common: CommonQueryParams, file: UploadFile = File(...)):
+async def transcribe(request: Request, common: CommonQueryParams, file: UploadFile = File(...)):
```

Comment:
Can you add a small decorator/dependency here that limits the size of the incoming audio packet? The limit we use in Flint is 10 MB: https://github.com/khoj-ai/flint/commit/b32c16be48be6f90c28f406416e8740b1ce0cd19#diff-8af27642228981ffa9a2c39239d7cbdfaa9f969f9c187be165c4cedc57b8c5f1R182-R194

Reply:
Good point. Added 10Mb file size limit allowed on the transcribe API endpoint